plugins/github-issues/api-report.md

```diff
@@ -0,0 +1,35 @@
+## API Report File for "@backstage/plugin-github-issues"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { RouteRef } from '@backstage/core-plugin-api';
+
+// Warning: (ae-forgotten-export) The symbol "GitHubIssuesProps" needs to be exported by the entry point index.d.ts
```

Comment:
Let's clean up this warning. Just nueed to export `GitHubIssuesProps` the entire way out of the package too.

Reply:
Done [ffd5e47](https://github.com/backstage/backstage/pull/12875/commits/ffd5e47fb5c6338aa0e6965d687d0028e0baabb5)